adage/analyze/api.py

```diff
@@ -255,8 +286,10 @@ def apply_filters(self, request, applicable_filters):
 
 
 class EdgeResource(ModelResource):
-    gene1 = fields.IntegerField(attribute='gene1_id', null=False)
-    gene2 = fields.IntegerField(attribute='gene2_id', null=False)
+    # gene1 = fields.IntegerField(attribute='gene1_id', null=False)
+    # gene2 = fields.IntegerField(attribute='gene2_id', null=False)
+    gene1 = fields.ForeignKey(GeneResource, "gene1", full=True)
+    gene2 = fields.ForeignKey(GeneResource, "gene2", full=True)
```

Comment:
I like this approach. Having written some front-end views, I think we should make use of `full=True` more often where it makes sense.